scripts/clean

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+rm -rf lib
```

Comment:
why is the "dist" directory called "lib"? b/c that's how you want it to look in the published, babel converted one.